lib/__tests__/cli.test.mjs

```diff
@@ -35,7 +35,6 @@ describe('buildCLI', () => {
 			cache: false,
 			color: false,
 			disableDefaultIgnores: false,
-			fix: false,
```

Comment:
`fix` default should remain `false`
see https://github.com/stylelint/stylelint/pull/8106/files#r1826592064

Reply:
see https://github.com/stylelint/stylelint/pull/8106/files#r1829415336